opensrp-chw/build.gradle

```diff
@@ -322,7 +322,7 @@ repositories {
 }
 
 dependencies {
-    implementation('org.smartregister:opensrp-client-chw-core:1.1.39-SNAPSHOT@aar') {
+    implementation('org.smartregister:opensrp-client-chw-core:1.1.39-DEMO-5-SNAPSHOT@aar') {
```

Comment:
Hey @Andati can you update this with a published version of the changes in this library?

Reply:
done